swagger/v2/move_include_parameter.yaml

```diff
@@ -9,15 +9,14 @@ MoveIncludeParameter:
     enum:
       - allocation
       - court_hearings
-      - documents
```

Comment:
Removed as not available in v2

Reply:
can you remove it from here: https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/644/files#diff-cd3b47805b696fdb1c1f6c2095866d53L38